packages/frontend/src/layouts/Navigation.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom'
+
+const list = [
+    { name: "home", path: "/" },
+    { name: "filmy", path: "/filmy" },
+    { name: "seriale", path: "/seriale" },
+]
+
+const Navigation: React.FC = () => {
+
+    const menu = list.map(item => (
```

Comment:
to nie musi byc nad returnem moze byc miedzy ul

Reply:
Wrzuciłam jako standardowe <NavLink> do ul i pozbyłam się map.